java/vector/src/main/java/org/apache/arrow/vector/ipc/ArrowFileWriter.java

```diff
@@ -73,8 +73,8 @@ public ArrowFileWriter(VectorSchemaRoot root, DictionaryProvider provider, Writa
 
   public ArrowFileWriter(VectorSchemaRoot root, DictionaryProvider provider, WritableByteChannel out,
                          Map<String, String> metaData, IpcOption option, CompressionCodec.Factory compressionFactory,
-                         CompressionUtil.CodecType codecType) {
-    super(root, provider, out, option, compressionFactory, codecType);
+                         CompressionUtil.CodecType codecType, int compressionLevel) {
```

Comment:
Well, now this is a breaking change, since the old constructor isn't available anymore.

Reply:
Updated